README.adoc

```diff
@@ -529,15 +529,15 @@ The more freedom we have, the more we want to be controlled by the SS again.
 
 And then we want our secret police back, and concentration death camps again, and the universal fear everywhere!
 
-Only then can we feel ourselves calm and secure.
+Only then can we feel calm and secure.
```

Comment:
This was taken from the YouTube translation, but OK, let's fix the translation here I guess.